fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java

```diff
@@ -523,86 +520,31 @@ public void write(DataOutput out) throws IOException {
     }
 
     public void readFields(DataInput in) throws IOException {
-        if (Catalog.getCurrentCatalogJournalVersion() < FeMetaVersion.VERSION_43) {
-            // consume the flag of empty user name
-            in.readBoolean();
-        }
-
-        // user name
-        if (Catalog.getCurrentCatalogJournalVersion() < FeMetaVersion.VERSION_30) {
-            qualifiedUser = ClusterNamespace.getFullName(SystemInfoService.DEFAULT_CLUSTER, Text.readString(in));
-        } else {
-            qualifiedUser = Text.readString(in);
-        }
-
-        if (Catalog.getCurrentCatalogJournalVersion() < FeMetaVersion.VERSION_43) {
-            int passwordLen = in.readInt();
-            byte[] password = new byte[passwordLen];
-            in.readFully(password);
-
-            // boolean isAdmin
-            in.readBoolean();
-
-            if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_1) {
-                // boolean isSuperuser
-                in.readBoolean();
-            }
-        }
+        qualifiedUser = Text.readString(in);
 
         if (Catalog.getCurrentCatalogJournalVersion() < FeMetaVersion.VERSION_100) {
```

Comment:
MINIMUM_VERSION_REQUIRED is 100, so I think we can remove this.

Reply:
Actually, I am removing code that FeMetaVersion < VERSION_96. I am afraid of version 100 is too latest.